vespajlib/src/main/java/ai/vespa/json/Json.java

```diff
@@ -211,6 +211,18 @@ public static Builder.Array existingSlimeArrayCursor(Cursor cursor) {
             if (cursor.type() != Type.ARRAY) throw new InvalidJsonException("Input is not an array");
             return new Builder.Array(cursor);
         }
+        public static Builder.Object fromObject(Json json) {
```

Comment:
There should be a from(Json) too, that delegates to one of Object or Array depending on the type?

Reply:
That method would have the return type `Builder` which does not provide any methods for mutating the structure.

Reply:
makes sense

Reply:
@jonmv `json.requireType(Type.OBJECT)` will throw `InvalidJsonException`

Reply:
Yes ... >_<